modules/storages/config/locales/en.yml

```diff
@@ -99,3 +99,4 @@ en:
     confirm_replace_oauth_client: "Are you sure? All users will have to authorize again against the storage."
     oauth_client_details_missing: "To complete the setup, please add OAuth client credentials from your storage."
     notice_oauth_application_replaced: "The OpenProject OAuth application was successfully replaced."
+    error_dependent_model_invalid: "Model is invalid!"
```

Comment:
Just from UX perspective, I'd have no idea, what this error wants to tell me, if I would see it when creating a storage.

Reply:
As the error will never be shown because it is already caught in the `StorageCreation` service model validation, there is in general no point in improving this any further.

@wielinde what do you think?

Reply:
You are both right. This whole PR is pretty hypothetical as by the time now we should not run into errors like those that we want to catch here. This PR stems from a situation where we did not do enough validations on the storage model. I don't have a better idea for the wording.

Reply:
We can improve this once we gained more experience with this. I suggest merging it nevertheless. Are you okay with that?

Reply:
fine for me